examples/flutter_gallery/.gitignore

```diff
@@ -1,6 +1,5 @@
 .DS_Store
 .atom/
-.idea
```

Comment:
You'll probably want to keep this exclusion. You can still commit individual files in `.dea/`, you just need to use `-f` when adding the file.